proposals/0002-lockout-violation-detection.md

```diff
@@ -0,0 +1,172 @@
+---
+simd: '0002'
```

Comment:
```suggestion
simd: '0009'
```

Reply:
it looks like the entire file should be renamed to `0009-lockout...` as well?

Reply:
Yes, that is correct.